bears/js/ESLintBear.py

```diff
@@ -58,15 +57,17 @@ def generate_config(filename, file):
         return '{"extends": "eslint:recommended"}'
 
     def process_output(self, output, filename, file):
-        if output[1]:
+        if not file or not output:
+            return
+
+        # ESLint throws error inside stdout
```

Comment:
what kind of errors? Would be good if you be as clear as possible here so people reading the code know what's meant ;)